app/src/features/surveys/components/MethodForm.tsx

```diff
@@ -72,7 +84,25 @@ export const SamplingSiteMethodYupSchema = yup.object({
           .typeError('End Date is required')
           .isValidDateString()
           .required('End Date is required')
-          .isEndDateSameOrAfterStartDate('start_date')
+          .isEndDateSameOrAfterStartDate('start_date'),
+        start_time: yup.string().nullable(),
+        end_time: yup
+          .string()
+          .nullable()
+          .test(
+            'checkDatesAreSameAndEndTimeIsAfterStart',
+            'End Date and Time must be after Start Date and Time',
```

Comment:
I actually think "End date and time must be after start date and time" is better than "End date and end time must be...", since the latter suggests that the end time must always be after the start time, which isn't necessarily true if the end date is after the start date.